src/mesh/impls/bout/boutmesh.cxx

```diff
@@ -844,13 +844,13 @@ int BoutMesh::load() {
   getCoordinates(CELL_CENTRE);
   if (StaggerGrids) {
     if (xstart >= 2) {
-      getCoordinates(CELL_XLOW);
+      //getCoordinates(CELL_XLOW);
     }
     if (ystart >= 2) {
       getCoordinates(CELL_YLOW);
     }
     if (LocalNz > 3) {
-      getCoordinates(CELL_ZLOW);
+      //getCoordinates(CELL_ZLOW);
```

Comment:
Why is this commented out?

Reply:
Sorry I shouldn't have left that in. Was a hack to get around an exception being thrown when interpolating to get the xlow coordinates, which I didn't actually want. Will fix...